### ISSUE TITLE
[Adaptive Cards - Home Page] - Screen reader is not announcing the role information for the 'Close (X)' icon control present inside the video player on the page.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9305]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 25H2 (OS Build 26200.5761)
Browser Version: Version 145.0.3734.1 (Official build) dev (64-bit)
Screen Reader: Narrator, NVDA

**Repro Steps:**

1. Turn on Narrator through Ctrl + Windows + Enter key.
2. Open the above URL on the Edge Dev browser.
3. Navigate to the 'Watch Video' control present below the 'A whole new way to deliver UI' heading text using tab key and activate using enter key.
4. Navigate to the 'Close (X)' icon control using tab key.
5. Verify that, screen reader is announcing the role information for the 'Close (X)' icon control present inside the video player on the page or not.

**Actual Result:**
Screen reader is not announcing the role information for the 'Close (X)' icon control present inside the video player on the page.

**Note**
Same issue replicates with NVDA.

**Expected Result:**
Screen reader should properly announce the role information as "button" for the 'Close (X)' icon control present inside the video player on the page.

**User Experience:**
Visually impaired users who rely on screen reader to access the web content will be impacted if screen reader is not announcing the role information for the 'Close (X)' icon control present inside the video player on the page due to which user will not be able to know the purpose, functionality of the control if it is not announced by screen reader.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/542689a1-a160-4511-a552-627bb516d67d

![Image](https://github.com/user-attachments/assets/2f214ea4-af8f-4a75-871c-1d8f4e305c5a)

### Card JSON

```JSON
NA
```
